src/components/TextField/TextField.story.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import storyWrapper from '../../storybook-addons/storyWrapper';
+import TextField from './TextField';
+import { TextFieldVariant } from './TextFieldEnums';
+import IconSearch from '../Icon/IconSearch';
+import IconNotification from '../Icon/IconNotification';
+
+const stories = storiesOf('TextField.TextField', module);
+
+stories.add(
+    'Overview',
+    storyWrapper(
+        'TextField is text input field.',
```

Comment:
Should this be "TextField is **a** text input field."?

Reply:
Thank you – well spotted